src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -168,11 +169,20 @@ class InstallAppsTask extends AbstractServerTask {
         looseWar.addManifestFile(manifestFile, "gradle-war-plugin")
     }
 
+    private boolean hasJavaSourceFiles(FileCollection classpath){
+        for(File f: classpath) {
+            if(f.getAbsolutePath().endsWith('/build/classes/java/main')) {
```

Comment:
If gradle allows you to configure the source to be someplace other than "java/main", Is is OK to just look for `contains('/build/classes/')`?   Also, I am concerned with this as a hardcoded string. Should we calculate this string from `project.buildDir + "/classes"`.  Better yet, is there a variable for the classes output location?